common/src/main/java/com/wynntils/core/text/StyledText.java

```diff
@@ -114,6 +120,10 @@ public static StyledText fromCodedString(CodedString codedString) {
         return fromString(codedString.getInternalCodedStringRepresentation());
     }
 
+    public String getString() {
+        return getString(PartStyle.StyleType.DEFAULT);
+    }
+
```

Comment:
We didn't need to add this or any change related to adding this. As the comment states on the original `getString` method, in an ideal world that method would be private..

Reply:
so, should i move the logic to StyedText if possible?